Operations.podspec

```diff
@@ -121,7 +129,47 @@ session Advanced NSOperations: https://developer.apple.com/videos/wwdc/2015/?id=
       'Sources/Extras/Contacts/iOS'
     ]
   end
-
+	
+  # Subspec which includes CloudKit functionality
+  s.subspec '+CloudKit' do |ss|
+    ss.platforms = { :ios => "8.0", :osx => "10.10" }
+    ss.dependency 'Operations/Standard'
+    ss.source_files = [
+      'Sources/Features/Shared/CloudCapability.swift',
+      'Sources/Features/Shared/CloudKitOperation.swift',			
+    ]
+  end
+	
+  # Subspec which includes CloudKit functionality
+  s.subspec '+CloudKit' do |ss|
+    ss.platforms = { :ios => "8.0", :osx => "10.10" }
+    ss.dependency 'Operations/Standard'
+    ss.frameworks = 'CloudKit'
+    ss.source_files = [
+      'Sources/Features/Shared/CloudCapability.swift',
+      'Sources/Features/Shared/CloudKitOperation.swift',			
+    ]
+  end
```

Comment:
This is a duplication in error I think which should be removed. 

Also CloudKit is available on tvOS platform, which will need to be added here, and updated in the excludes from Standard spec.